File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    int answer = 0;
    //
    int a_ = stoi(to_string(a) + to_string(b));
    int b_ = stoi(to_string(b) + to_string(a));
    answer = max(a_, b_);
    //
    return answer;
}

File: 프로그래머스/0/181939. 더 크게 합치기/더 크게 합치기.java
import java.util.*;

class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        Integer q = Integer.parseInt(String.valueOf(a) + String.valueOf(b));
        Integer w = Integer.parseInt(String.valueOf(b) + String.valueOf(a));
        if (q > w) {
            answer = q;
        } else {
            answer = w;
        }
        return answer;
    }
}